packages/demo/src/components/examples/BadgeExamples.tsx

```diff
@@ -9,6 +9,10 @@ export class BadgeExamples extends React.Component<any, any> {
                     <label className="form-control-label">Default Badge</label>
                     <div className="form-control">
                         <Badge label="Badge label" />
+                        <Badge label="Badge critical" extraClasses={['mod-critical']} />
+                        <Badge label="Badge warning" extraClasses={['mod-warning']} />
+                        <Badge label="Badge information" extraClasses={['mod-information']} />
+                        <Badge label="Badge success" extraClasses={['mod-success']} />

```

Comment:
Could we have some margin-x for these examples? They sit so close :joy:  :socialdistancing:
![image](https://user-images.githubusercontent.com/52677246/105899097-3d464980-5fe8-11eb-89a4-2d47dfca7f88.png)

Reply:
man you are faster than me xD 
![image](https://user-images.githubusercontent.com/63734941/105900456-10933180-5fea-11eb-9ed1-d61fcc708241.png)

Reply:
Will the badge ever be stacked on one another? :thinking:  If so we need maybe a m1? :eyes: :nerd_face: